### ISSUE TITLE
Typing image descriptions is very laggy

### ISSUE THREAD

**Issue by averageunicorn** [tuskyapp/Tusky#544]
When I'm writing a caption, it takes seconds for the text to actually appear. I think it might be related to image size, as it didn't happen when I tested it with a small graphic right now, but happened again when I tried it with a photo.

Version 1.5.2, Android 7.1.2, using Gboard.

**Comment by charlag** [comment]
Yeah, image actually resizes when you type the text so with large image it make take some time, it could be improved, thanks!

**Comment by charlag** [comment]
I assign it low priority because I've heard no other complains

**Comment by connyduck** [comment]
There was an issue where Tusky used way too much memory when openening the caption dialog that has been resolved. It may very well been the cause for your issue. Can you try it with Tusky 3.0 and report if it still persists?
